openqa-investigate

```diff
@@ -119,6 +129,37 @@ trigger_jobs() {
     fi
 }
 
+check_dependencies_to_determine_whether_to_skip_or_postpone() {
+    local id=$1 job_data=$2 parallel_parents dependency_data pending_cluster_jobs cluster_jobs relevant_cluster_results
+
+    # skip parallel children (will be handled together with parallel parent)
+    # note: For this to work, this script needs to be executed for all job results (so we won't miss the investigation in case the parallel parent has passed).
```

Comment:
Looks like that's currently not possible unless one would list all possible job results in the config (to have a line like `job_done_hook_enable_failed = 1` for every job result).